theano/gpuarray/opt.py

```diff
@@ -200,6 +204,8 @@ def op_lifter(OP, cuda_only=False):
     """
 
     def f(maker):
+        @register_topo('fast_compile')
+        @local_optimizer(OP)
```

Comment:
Why this here? I don't think this is good. Sometimes we will want to pass extra parameter to local_optimizer and register_topo.